api/message.go

```diff
@@ -130,6 +130,10 @@ func (a *MessageAPI) CreateMessage(ctx *gin.Context) {
 	message := model.Message{}
 	if err := ctx.Bind(&message); err == nil {
 		message.ApplicationID = a.DB.GetApplicationByToken(auth.GetTokenID(ctx)).ID
```

Comment:
It would be better if `a.DB.GetApplicationByToken` would be executed only once here, because it makes an call to the database.